src/parser/core/Events.ts

```diff
@@ -455,7 +469,7 @@ export interface HealEvent extends Event<EventType.Heal> {
   /** If the event is a tick of a HoT or HoT like object */
   tick?: boolean;
   resourceActor: number;
```

Comment:
```suggestion
  resourceActor: ResourceActor;
```
should be able to set the type of these fields to `ResourceActor` since you set the numeric enum values, right?

Reply:
.. oh probably, I think I hesitated because I wasn't sure what would happen if the raw data gave us a number that wasn't expected by the enum